packages/astro/test/fixtures/astro-global/src/pages/index.astro

```diff
@@ -5,6 +5,6 @@
 </head>
 <body>
   <div id="pathname">{Astro.request.url.pathname}</div>
-  <a id="site" href={Astro.site.origin}>Home</a>
+  <a id="site" href={Astro.site.toString()}>Home</a>
```

Comment:
I'm curious whether a simple `Astro.site` works here?

Reply:
Probably a bug if not! I'll update it.

Reply:
Updated this, thanks for the suggestion